examples.json

```
@@ -0,0 +1,347 @@
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "400": { "description": "Invalid ID supplied" },
+          "404": { "description": "Bookmark not found" }
+        },
+        "security": [
+          { "bookmarks_auth": ["write:bookmarks", "read:bookmarks"] }
+        ]
+      }
+    }
+  },
+  "parameters": {
+    "BearerHeader": {
+      "in": "header",
+      "name": "Authorization",
+      "description": "JWT token required to get resources.",
+      "type": "string",
+      "required": true,
+      "default": "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxIiwibmFtZSI6IkEiLCJpYXQiOjEsImVic2NvX3B1YV9pZCI6IjA3MjQyZjlhIn0.nRe7fIjMAIf1SkpFOTAEga3Gxx_Tk_QUgwATrgZpGfg"
+    },
+    "PersonalizedHeader": {
+      "in": "header",
+      "name": "X-EIS-ID-TOKEN",
+      "description": "Service identity header.",
+      "type": "string",
+      "required": true,
+      "default": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxIiwibmFtZSI6IkEiLCJpYXQiOjEsImVic2NvX3B1YV9pZCI6IjA3MjQyZjlhIn0.nRe7fIjMAIf1SkpFOTAEga3Gxx_Tk_QUgwATrgZpGfg"
+    }
+  },
+  "securityDefinitions": {
+    "bookmarks_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://bookmarksstore.swagger.io/oauth/authorize",
+      "flow": "implicit",
+      "scopes": {
+        "write:bookmarks": "modify bookmarks in your account",
+        "read:bookmarks": "read your bookmarks"
+      }
+    },
+    "api_key": { "type": "apiKey", "name": "api_key", "in": "header" }
+  },
+  "definitions": {
+    "BookmarkItem": {
+      "type": "object",
+      "properties": {
+        "an": {
+          "type": "string",
+          "example": "vmc.b30177686"
+        },
+        "contributors": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Contributors"
+          }
+        },
+        "shortDBName": {
+          "type": "string"
+        },
+        "title": {
+          "$ref": "#/definitions/Title"
+        },
+        "recordFields": {
+          "type": "object"
+        }
+      }
+    },
+    "BookmarkRecord": {
+      "type": "object",
+      "required": ["notes", "record", "tags"],
+      "properties": {
+        "dateCreated": {
+          "type": "string",
+          "example": "2019-04-22T12:35:25.891Z"
+        },
+        "dateModified": {
+          "type": "string",
+          "example": "2019-04-22T12:35:25.891Z"
+        },
+        "id": {
+          "type": "string"
+        },
+        "notes": {
+          "type": "string"
+        },
+        "record": {
+          "$ref": "#/definitions/BookmarkRecord"
+        },
+        "tags": {
+          "$ref": "#/definitions/Tags"
+        }
+      }
+    },
+    "Category": {
+      "type": "object",
+      "properties": {
+        "id": { "type": "integer" },
+        "name": { "type": "string" }
+      },
+      "xml": { "name": "Category" }
+    },
+    "Contributors": {
```

This should be singular `Contributor`, I think.
```suggestion
    "Contributor": {
```